lib/pbench/server/api/__init__.py

```diff
@@ -33,6 +33,7 @@
     SampleValues,
 )
 from pbench.server.api.resources.query_apis.datasets_search import DatasetsSearch
+from pbench.server.api.resources.quisby import QuisbyData
```

Comment:
Shall we only name this to Quisby or maybe PQuisby?

Reply:
That's an interesting question I hadn't really thought about. Quisby is the package we're using to support visualization and comparison ... but I have a feeling we should be choosing something more descriptive and less "brand namey" for the actual interface and Pbench implementation, like "visualize".

I think maybe that's complicated semantically by the fact that the server isn't "visualizing" but rather using Quisby to postprocess the CVS files into a form the client (e.g., dashboard) can easily use to visualize ... but we have the API to *support* client visualization, not "to Quisby". 🤔